lib/model/content.dart

```diff
@@ -253,17 +253,14 @@ class HeadingNode extends BlockInlineContainerNode {
 
 enum ListStyle { ordered, unordered }
```

Comment:
This enum is no longer really pulling its weight, as it's redundant with the two subclasses; let's cut it out.

The remaining references to it, all within `parseListNode`, can just refer to `element.localName` directly.